src/service/resources-impl.js

```diff
@@ -2278,31 +2278,33 @@ export class Resources {
       doPass();
     };
 
-    vsm.addTransition(prerender, prerender, doPass);
-    vsm.addTransition(prerender, visible, doPass);
-    vsm.addTransition(prerender, hidden, doPass);
-    vsm.addTransition(prerender, inactive, doPass);
-    vsm.addTransition(prerender, paused, doPass);
-
-    vsm.addTransition(visible, visible, doPass);
-    vsm.addTransition(visible, hidden, doPass);
-    vsm.addTransition(visible, inactive, unload);
-    vsm.addTransition(visible, paused, pause);
-
-    vsm.addTransition(hidden, visible, doPass);
-    vsm.addTransition(hidden, hidden, doPass);
-    vsm.addTransition(hidden, inactive, unload);
-    vsm.addTransition(hidden, paused, pause);
-
-    vsm.addTransition(inactive, visible, resume);
-    vsm.addTransition(inactive, hidden, resume);
-    vsm.addTransition(inactive, inactive, noop);
-    vsm.addTransition(inactive, paused, doPass);
-
-    vsm.addTransition(paused, visible, resume);
-    vsm.addTransition(paused, hidden, doPass);
-    vsm.addTransition(paused, inactive, unload);
-    vsm.addTransition(paused, paused, noop);
+    const addTransition = vsm.addTransition.bind(vsm); // for size savings.
```

Comment:
Small edits like this are not worthwhile, and are sometimes detrimental. This change actually _increased_ the brotli size by 90 bytes!

Don't make changes to code because of repetition, trust the compressor to compress it for you. Only make changes like this if it makes the code easier to understand. Or, if it improves performance (this code creates a function allocation, and goes through a bunch of unnecessary spec edge cases). In this case, we've regressed on all three.

Reply:
I was going by the gzip size, which is what we check for. Will undo.